src/components/register/steps/ticket/day.tsx

```diff
@@ -23,9 +24,15 @@ const Grid = styled.div`
 
 const TicketDay = (_: RouteComponentProps) => {
 	const { eventStartDate, eventEndDate } = useSiteMetadata()
-	const { register, handleSubmit } = useForm<{ day: string }>()
+	const { register, watch, handleSubmit } = useForm<{ day: string }>()
 	const dispatch = useAppDispatch()
 
+	useEffect(() => {
+		const subscription = watch(data => dispatch(ChangeTicketDay.create(data.day)))
+
+		return () => subscription.unsubscribe()
```

Comment:
Since we need to return a function, can we not just return `subscription.unsubscribe`?